libs/ledger-live-common/src/hw/getBatteryStatus.ts

```diff
@@ -22,38 +22,55 @@ enum FlagMasks {
   ISSUE_TEMPERATURE = 0x00000020,
 }
 
-const getBatteryStatus = async (
+// conditional type used to infer what will be the tuple of returned values of the getBatteryStatus
+// command. According to the tuple that of StatusTypes that is requested, it will infer the corresponding
```

Comment:
`According to the tuple of BatteryStatusTypes that is requested` no ?

Reply:
yep, good catch, done.